<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_Table_Dialog</class>
 <widget class="QDialog" name="Create_Table_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 20pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Create Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Table Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_create_table_dialog">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>160</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel_create_table_input">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_elements_to_table">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add elements to table</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_admin_menuebar"/>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current time</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_create_table_dia_columns">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
